src/openrct2-ui/WindowManager.cpp

```diff
@@ -417,6 +417,7 @@ class WindowManager final : public IWindowManager
                 gToolbarDirtyFlags |= BTM_TB_DIRTY_FLAG_PEEP_COUNT;
                 window_invalidate_by_class(WC_GUEST_LIST);
                 window_invalidate_by_class(WC_PARK_INFORMATION);
+                window_guest_list_refresh_list();
```

Comment:
Is it necessary/useful to have this function separate from the invalidation function?

Reply:
And why not `window_refresh_guest_list`?

Reply:
Yes, invalidate is for forcing a redraw this is forcing a recalc of the data. Calling this function on every invalidate would be expensive (i tasted that already). I guess you could do the reverse invalidate on all calls to this function but not really a huge need.